src/js/smooch.jsx

```diff
@@ -58,7 +58,7 @@ function renderLink() {
 function getDeviceId() {
     const SK_STORAGE = 'sk_deviceid';
     const deviceId = storage.getItem(SK_STORAGE) ||
-        uuid.v4().replace(/-/g, '');
+    uuid.v4().replace(/-/g, '');
```

Comment:
I think keep using latest esformatter, we'll just tolerate the fucky indent for now. or save without esformatter if it really bothers you